django/thunderstore/repository/comment.py

```diff
@@ -0,0 +1,80 @@
+from django import forms
+from django.contrib.auth.models import User
+from django.core.exceptions import ValidationError
+
+from thunderstore.community.models import PackageListing
+from thunderstore.repository.models import Comment
+
+
+def clean_content(content: str) -> str:
+    return content.strip()
+
+
+class CreateCommentForm(forms.ModelForm):
+    class Meta:
+        model = Comment
+        fields = ["content"]
+
+    def __init__(
+        self,
+        user: User,
+        commented_object: PackageListing,
```

Comment:
the type annotation here should probably just be `Model`, or alternatively the form should probably validate the `commented_object` is actuall of instance `PackageListing`, right?

Reply:
Actually I think validating the model is a `PackageListing` could be a good idea either way for now, given that the db schema allows it to be any model and we don't want to accidentally shove comments elsewhere just yet

Reply:
Using a `Union` (in the future) with `PackageListing` would be best I think as:
- It makes it clear to readers what parents are supported
- If you need to special case every possible parent type, type checking can help check for unhandled cases